src/search.py

```diff
@@ -188,7 +188,7 @@ def get_webhelp_page_index_objects(content: Tag, url: str, page_path: str, title
             chapter_title_node = chapter.select_one('[data-toc]')
             if chapter_title_node:
                 chapter_title = chapter_title_node.extract().text
-                chapter_title_anchor = chapter_title_node.attrs["data-toc"].split('#')[1]
```

Comment:
Anchors in right ToC had these format:
`data-toc="multiplatform#kotlin-multiplatform-use-cases"`
And now they look like:
`data-toc="kotlin-multiplatform-use-cases"`
So, splitting by # is obsolete.